versioned_docs/version-stable/quickstart/nym-connect.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_label: "Nym Connect"
+description: "Nym is a blockchain-based privacy platform. It provides strong network-level privacy against sophisticated end-to-end attackers, and anonymous transactions using blinded, re-randomizable, decentralized credentials."
+hide_title: false
+---
+
+# Nym Connect
+
+Follow this guide to install and run Nym connect.
+
+### Step 1. Download
+
+Get the latest release of nym connect app [here](https://github.com/nymtech/nym/releases/tag/nym-connect-v1.0.2).
+
+:::note
+Nym connect build for linux currently supports only **debian**. If you wish to run on another distro that is not debian, you'll need to build from source. To build from source, you'll also need a [Developer ID Application](https://developer.apple.com/support/developer-id/).
```

Comment:
Its not that NymConnect only supports debian, its that the pre-compiled binaries are built for debian-based systems, and if you aren't running on a debian-based flavour of linux you'll need to compile it yourself